src/wmnds/patterns/content-cards/email-sign-up/_email-sign-up.njk

```diff
@@ -0,0 +1,66 @@
+{% from "wmnds/components/button/_button.njk" import wmnButton %}
+{% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
+{# Set #}
+{% set formClass = " wmnds-is--open" if params.expanded else "" %}
+{# Button variables #}
+{% set btnText = "Sign up for updates" if params.expanded or params.externalLink else "Continue" %}
+{% set btnClass = " wmnds-btn--primary" if not params.externalLink else ""  %}
+{% set btnSpacingClass = " wmnds-m-t-xsm" if not params.expanded else ""  %}
+{% set btnType = "type=\"submit\"" if params.expanded or params.externalLink else "" %}
+{# Create a string where we can use the nunjucks '| replace' #}
+{% set inputId = "esu-form-%%" + ( "-" + params.exampleSuffix or "") %}
+
+{% call wmndsContentCard() %}
+    <div class="wmnds-p-sm wmnds-col-1">
+        <h2 class="wmnds-content-card__title">Stay up to date</h2>
+        <p class="wmnds-m-b-md">Receive an email notification when this information changes.</p>
+        {#- External link -#}
+        {% if params.externalLink %}
+        <a href="#" target="_blank" class="wmnds-btn {{btnClass}} wmnds-col-1">{{btnText}}</a>
+        {%- else %}
+        <form class="js-cm-form wmnds-content-card__form{{formClass}}" id="subForm-{{params.exampleSuffix}}" action="https://www.createsend.com/t/subscribeerror?description=" method="post" data-id="CREATE_SEND_DATA_ID">
+            <div class="wmnds-m-b-sm">
+                {%- set emailInputId = inputId | replace('%%', 'email') %}
+                <label class="wmnds-fe-label" for="{{emailInputId}}">Email address</label>
+                <input class="js-cm-email-input qa-input-email wmnds-fe-input" id="{{emailInputId}}" name="{{emailInputId}}" required="" type="email"/>
+            </div>
+            <!-- Expanding content -->
+            <div class="wmnds-content-card__form-collapse">
+                <div class="wmnds-m-b-sm">
+                    {%- set nameInputId = inputId | replace('%%', 'name') %}
+                    <label class="wmnds-fe-label" for="{{nameInputId}}">Name</label>
+                    <input class="wmnds-fe-input" id="{{nameInputId}}" name="{{nameInputId}}" maxlength="200" type="text"/>
+                </div>
+                <div class="wmnds-m-b-sm">
+                    {%- set postcodeInputId = inputId | replace('%%', 'postcode') %}
+                    <label class="wmnds-fe-label" for="{{postcodeInputId}}">Postcode</label>
+                    <p class="wmnds-m-b-sm">We'll use this to send relevant emails to your location</p>
+                    <input class="wmnds-fe-input" id="{{postcodeInputId}}" name="{{postcodeInputId}}" maxlength="200" required="" type="text"/>
+                </div>
+                <label class="wmnds-fe-checkboxes__container wmnds-m-t-md">
+                    I agree to WMCA's Privacy Policy
+                    <input class="wmnds-fe-checkboxes__input" required="true" aria-required="true" checked="true" type="checkbox"/>
```

Comment:
The form inputs need to use the same classes/ids/names etc. that the campaign monitor snippet uses otherwise it won't work when submitted to campaign monitor. If you try submit the form now you'll notice it errors out.

Just add the design system classes to the inputs/form etc. so that it is styled correctly but under the hood it should all be the campaign monitor code. You can change a lot of the code other than the attributes on the inputs and form as I believe this is what campaign monitor "scans" for when you submit.

```html
<div>
  <h1 color="#000">Get the latest updates from us</h1>
  <form class="js-cm-form" id="subForm" action="https://www.createsend.com/t/subscribeerror?description=" method="post" data-id="92D4C54F0FEC16E5ADC2B1904DE9ED1A99364085B43530088A4D2FC306FC12C93064C1C79B7ABC5EE4710A358CB5FBFB9279A45ECC576F3F096FDC1EFB755F64">
    <div>
      <div>
        <label>Name </label>
        <input aria-label="Name" id="fieldName" maxlength="200" name="cm-name">
      </div>
      <div>
        <label>Email </label>
        <input autocomplete="Email" aria-label="Email" class="js-cm-email-input qa-input-email" id="fieldEmail" maxlength="200" name="cm-nllddr-nllddr" required="" type="email">
      </div>
      <div>
        <label>Home Postcode </label>
        <input aria-label="Home Postcode" id="fieldkdyhlr" maxlength="200" name="cm-f-kdyhlr" required="">
      </div>
      <div>
        <div>
          <div>
            <input aria-required="" id="cm-privacy-consent" name="cm-privacy-consent" required="" type="checkbox"><label for="cm-privacy-consent">I agree to be emailed</label>
          </div>
          <input id="cm-privacy-consent-hidden" name="cm-privacy-consent-hidden" type="hidden" value="true">
        </div>
      </div>
    </div>
    <button type="submit">Subscribe</button>
  </form>
</div>
`

Reply:
I've added the markup from campaign monitor but to avoid accessibility errors and console errors i've given each id a suffix and have included a note on the page for the reason why